### ISSUE TITLE
sample code to copy data from a table in webpage into a spreadsheet?

### ISSUE THREAD

**Issue by jerronl** [xlwings/xlwings#2511]
Do we have sample code to sample code to copy data from a table in webpage into a spreadsheet? Even better if it is with selenium.
Thanks!

**Comment by fzumstein** [comment]
No, but maybe have a look at https://pandas.pydata.org/docs/reference/api/pandas.read_html.html ?
